drivers/power/adp1050/adp1050.h

```diff
@@ -328,6 +374,11 @@
 #define ADP1050_PWM_180_PHASE_SHIFT_SETTINGS	0xFE3B
 #define ADP1050_MODULATION_LIMIT		0xFE3C
 #define ADP1050_FEEDFORWARD_SS_FILTER_GAIN	0xFE3D
+/***********Added Support for ADP1051******************/
```

Comment:
i think `ADP1051_` prefix should suffice. These comments are unnecessary.

Reply:
copy